doc/source/configuration.rst

```diff
@@ -3,112 +3,294 @@
+This section describes the different possibilities to adapt the application to work with various data structures or
+even custom data sources. If you are planning to implement such modifications, we suggest to check all possible
 solutions first, as the necessary effort can vary significantly depending on your specific needs.
 
+.. _configuration-initial-setup:
+
+Create the inital database setup
+--------------------------------
+
+Out of the box the pyramid_oereb server supports three different topic configurations in the database:
+
+  - the **pyramid_oereb standard model**
+  - the **interlis 2.3 OeREBKRM transfer model**
+  - the **OEREBlex model**
+
+Pyramid_oereb Standard Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This schema and table structure is based on the initial topic structure used in the pyramid_oereb
+v1.x versions. It's mainly used for cantonal topics which are not (yet) stored in the interlis 2.3 OeREBKRM-
+Transfer structure and by cantons that do not use OeREB-Lex to manage the legal documents.
+
+Interlis 2.3 OeREBKRM Transfer Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All the federal data sets are provided in this data structure. So this is the schema and table model you
+want to use for all the federal topics unless you want to transform the data to a specific database structure.
+If your cantonal data is also stored based on this model, then you probably want to use this structure 
+for all topics to homogenize your database content.
+The `Ili2pg Oereb Data Import Manual <https://github.com/openoereb/ili2pg_oereb_data_import_manual>`__
+explains how to use the ili2pg tool to create the corresponding schema and how to import the XML data.
+
+
+OEREBlex Topic Model
+^^^^^^^^^^^^^^^^^^^^
+
+This third model is usefull if you maintain your legal documents using the OEREBlex application and you
+have a specific cantonal model for your data. It is similar to the pyramid_oereb standard model, but all
+the document related tables are omitted. Instead the documents are linked by the geolink attribute. 
 
 .. _configuration-additional-topics:
 
-Add additional topics
----------------------
+Add additional standard topics
+------------------------------
+
+If you like to add one or more additional topics based on the *pyramid_oereb standard* database structure
+you can use the internal command below creating an SQL script to establish the topic schema.
 
-If you only need to add one or more additional cantonal topics and want to use the standard database structure
-for these too, you can run a provided script to generate the required models.
+But before creating any new topic structure you have to add its configuration. Open the configuration file
+(pyramid_oereb.yml) and copy the section from one of the existing **standard** topics which usually 
+looks like this:
+
+.. code-block:: yaml
+
+    - code: ch.NE.Baulinien
+      geometry_type: LINESTRING
+      thresholds:
```

Comment:
The threshold section should be reduced, since some parameters are not supported anymore.
[Since we have updated the [gitlab Readme](https://github.com/openoereb/pyramid_oereb/blob/master/README.rst) to ](https://github.com/openoereb/pyramid_oereb/blob/master/doc/source/changes.rst#theme-configuration)

Reply:
@svamaa: Adapted as suggested.